src/jit/emitxarch.cpp

```diff
@@ -11196,9 +11196,21 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
     assert(sz == emitSizeOfInsDsc(id));
 
 #if !FEATURE_FIXED_OUT_ARGS
+    bool bUpdateStackLv = true;
+
+    if (emitIGisInProlog(ig))
+        bUpdateStackLv = false;
+    if (emitIGisInEpilog(ig))
+        bUpdateStackLv = false;
+#if FEATURE_EH_FUNCLETS
+    if (emitIGisInFuncletProlog(ig))
+        bUpdateStackLv = false;
+    if (emitIGisInFuncletEpilog(ig))
+        bUpdateStackLv = false;
+#endif // FEATURE_EH_FUNCLETS
```

Comment:
@janvorli @BruceForstall

Is it required to add ``_TARGET_X86_``? This path affects only x86 ports as ``!FEATURE_FIXED_OUT_ARGS`` is true only for x86. In addition, ``FEATURE_EH_FUNCLETS`` is true only for x86/Linux with RyuJIT. Thus, there is no effect on Windows x86 even in the current form.

In addition, this patch seems to be not related with architecture. The current implementation of ``emitter::emitBegProlog`` does not allow the stack level adjustment in prolog/epilog, and thus the same issue will occur when any windows port takes this code path.

I would like not to append ``_TARGET_X86_`` if it has no issue, but please let me know your opinion.

For the code style, I'll revise as you suggested.